core/requires.js

```diff
@@ -29,6 +29,7 @@ goog.provide('Blockly.requires');
 
 // Blockly Core.
 goog.require('Blockly');
+goog.require('Blockly.Comment');
```

Comment:
Might be nice to add a comment that comments should be disabled in the inject options if removing this: https://developers.google.com/blockly/guides/get-started/web#configuration

Reply:
Done